### PR TITLE
Adds new rubocops.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made? 🤔
All the rubos.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, including data writes to shared file systems, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: app/reports/invalid_form_uris.rb [hunk 1]
```diff
@@ -5,14 +5,14 @@
 class InvalidFormUris
   JSON_PATH = '$.**.form.uri'
   SQL = <<~SQL.squish.freeze
-    SELECT (jsonb_path_query_array(description, '#{JSON_PATH} ? (@ like_regex "^.*\.html$")') ||
```
Comment (ndushay) [inline app/reports/invalid_form_uris.rb:8]: actually, I think the period here is meant to be a real period, not an anything char.

File: app/reports/invalid_form_uris.rb [hunk 2]
```diff
@@ -5,14 +5,14 @@
...
+    SELECT (jsonb_path_query_array(description, '#{JSON_PATH} ? (@ like_regex "^.*.html$")') ||
             jsonb_path_query_array(description, '#{JSON_PATH} ? (@ like_regex "^(?!https?://).*$")')) ->> 0 as value,
            external_identifier,
            jsonb_path_query(identification, '$.catalogLinks[*] ? (@.catalog == "symphony").catalogRecordId') ->> 0 as catkey,
            jsonb_path_query(structural, '$.isMemberOf') ->> 0 as collection_id
            FROM "dros" WHERE
            (jsonb_path_exists(description, '#{JSON_PATH} ? (@ like_regex "^(?!https?://).*$")') OR
-           jsonb_path_exists(description, '#{JSON_PATH} ? (@ like_regex "^.*\.html$")'))
```
Comment (ndushay) [inline app/reports/invalid_form_uris.rb:15]: ditto here...

File: app/reports/invalid_language_uris.rb [hunk 3]
```diff
@@ -7,18 +7,18 @@ class InvalidLanguageUris
...
             jsonb_path_query_array(description, '#{JSON_PATH2} ? (@ like_regex "^(?!https?://).*$")')) ->> 0 as contrib,
            external_identifier,
            jsonb_path_query(identification, '$.catalogLinks[*] ? (@.catalog == "symphony").catalogRecordId') ->> 0 as catkey,
            jsonb_path_query(structural, '$.isMemberOf') ->> 0 as collection_id
            FROM "dros" WHERE
            (jsonb_path_exists(description, '#{JSON_PATH1} ? (@ like_regex "^(?!https?://).*$")') OR
-           jsonb_path_exists(description, '#{JSON_PATH1} ? (@ like_regex "^.*\.html$")') OR
+           jsonb_path_exists(description, '#{JSON_PATH1} ? (@ like_regex "^.*.html$")') OR
            jsonb_path_exists(description, '#{JSON_PATH2} ? (@ like_regex "^(?!https?://).*$")') OR
-           jsonb_path_exists(description, '#{JSON_PATH2} ? (@ like_regex "^.*\.html$")')
```
Comment (ndushay) [inline app/reports/invalid_language_uris.rb:21]: and here

**Review by ndushay — CHANGES_REQUESTED** [review]
I think the "\." in these cases are intentional - it should be a period, not an anything char?

**Comment by justinlittman** [comment]
Great catch @ndushay! I excluded that rule for all reports.

**Review by ndushay — APPROVED** [review]

**Pull request merged**